.gitattributes

```diff
@@ -20,3 +20,5 @@ app/_locales/** linguist-generated
 # yarn berry suggested .gitattributes
 /.yarn/releases/** binary
 /.yarn/plugins/** binary
+
+app/*.html linguist-language=EJS
```

Comment:
This is supposed to make GitHub UI treat our `app/*.html` files as if they were EJS templates, but it doesn't seem to be working?

docs are here: https://github.com/github-linguist/linguist/blob/master/docs/overrides.md#using-gitattributes

The docs mention:

> the added attributes will not take effect until the .gitattributes file is committed to your repository.

And maybe that actually means it needs to be committed to the default branch to take effect?

Anyone know what's i'm doing wrong, if anything?